bin/cs

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+CACHE=".php_cs_cache"
+
+if [[ ! -d ${CACHE} ]] ; then
```

Comment:
i believe, at this point, im quoting everything :P `-d "${..}"` and `mkdir "${...}"`.

Reply:
Erm … Nope, they're not. I've updated the ones in this file though.